.travis.yml

```diff
@@ -20,7 +20,7 @@ before_install:
  - travis_retry sudo apt-get update -qq
  - travis_retry sudo apt-get install -qq python-pygments qt5-default qt5-qmake qtbase5-dev qtcreator libxml2-utils libpcre3 gdb unzip wx-common xmlstarlet
  - travis_retry sudo python2 -m pip install --upgrade pytest==4.6.4
- - travis_retry python2 -m pip install pylint
+ - travis_retry python2 -m pip install --user pylint
```

Comment:
Why it should be installed with `--user` option?
I would also suggest to explicitly specify `pylint` version to avoid possible compatibility problems like [this](https://github.com/danmar/cppcheck/pull/1996#issuecomment-511763501).

Reply:
`--user` worked out unless another prior attempt.

For now I am satisfied with the result...

Reply:
In the previous attempt there was a [conflict](https://travis-ci.org/danmar/cppcheck/jobs/560141122#L729) with enum34 module which was installed with apt-get as dependency, I suppose. Here is some explanation: https://askubuntu.com/questions/1050158/problem-installing-tensorflow.

Reply:
I think we'd better use `--user` in travis for other pip dependencies to avoid same problems, including `pytest` installation.